enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/document/DocumentProcessingService.java

```diff
@@ -199,9 +198,16 @@ public void checkDocumentSubmitWithProvider(OwnerId ownerId, DocumentResultEntit
             docSubmitResult.setErrorDetail(e.getMessage());
         }
 
-        documentResultEntity.setErrorDetail(docSubmitResult.getErrorDetail());
+        if (docSubmitResult.getErrorDetail() != null) {
```

Comment:
Maybe we should use `StringUtils.isNotBlank` to be consistent with other occurrences.